datadog/resource_datadog_user.go

```diff
@@ -90,75 +118,244 @@ func buildDatadogUserStruct(d *schema.ResourceData) *datadog.User {
 	return &u
 }
 
+func buildDatadogUserV2Struct(d *schema.ResourceData, providerConf *ProviderConfiguration) *datadogV2.UserCreateRequest {
+	userAttributes := datadogV2.NewUserCreateAttributesWithDefaults()
+	userAttributes.SetEmail(d.Get("email").(string))
+	userAttributes.SetName(d.Get("name").(string))
```

Comment:
If this is optional now, will if fail if not set? Should we GetOk instead

Reply:
I don't think it will fail, but it will set it to the empty string instead of null. Using GetOk.